pages/members/[slug].js

```diff
@@ -177,12 +203,57 @@ const MemberPage = ({ member, loadError }) => {
                 </Linkify>
               </p>
             }
+            </div>
+
+      <div className="flex flex-col items-start md:w-2/3">
+        <div>
+        <div className="page-title flex justify-between">
+          <h3 className="mt-16 mb-4">Upcoming events {member.screenname} is going to:</h3>
+          {/* <div className="action">
+            { member && member.roles.includes('event-creator') &&
+              <Link href="/events/create">
+                <a className="btn-primary">Create event</a>
+              </Link>
+            }
+          </div> */}
+        </div>
+        
+          <UpcomingEvents
+          allowCreate
+          limit={ 30 }
+          page={ 1 }
+          labelLink={null}
+        />
+        </div>
+        
+        <div className="flex flex-col">
+          <div className="flex flex-col items-start mb-10">
+             <p className='font-semibold text-md mt-8'>Stay Social</p>
+             <ul className='space-y-1 mt-4'>
+               {links ? links.map((link) => {
+               <li><a href={link.url}>{link.name}</a></li>
+               })
+               : 'No links yet'
+               }
+             </ul>
           </div>
+          <div className="flex items-start mb-10 border border-line p-4 w-fit">
+              <form className='flex flex-col space-y-6 w-96' onSubmit={() => platform.user.patch(member._id, { links: [ { name: name, url: url } ] })}>
```

Comment:
This needs to be taken out into a function, and you'll need to make it possible to add multiple links.